packages/coinjoin/src/client/round/selectRound.ts

```diff
@@ -135,8 +135,46 @@ export const getAccountCandidates = ({
             return [];
         }
 
+        // filter out InputLongBanned utxos until we know how to deal with them...
+        const [_, whitelistedUtxos] = arrayPartition(
+            account.utxos,
+            utxo =>
+                prison.isDetained(utxo.outpoint)?.errorCode ===
+                WabiSabiProtocolErrorCode.InputLongBanned,
+        );
```

Comment:
why `arrayPartition` instead of basic filter?
```suggestion
        const whitelistedUtxos = account.utxos.filter(
  (utxo) =>
    prison.isDetained(utxo.outpoint)?.errorCode !==
    WabiSabiProtocolErrorCode.InputLongBanned
);
```

Reply:
see comment:
```
// filter out InputLongBanned utxos until we know how to deal with them...
```

currently they are stored in `_` until we know :)

Reply:
I see :)